### PR TITLE
#832: Index of repos for each Kubeflow Docker Image

### PR DESCRIPTION
**Pull request by JavaDerek**
The City Scholars from would like to share 2 pieces of documentation for review.  The first is a table showing the GitHub repos for each Docker image we have seen running as a part of Kubeflow.  This is very useful to us because we have to have all Docker images security scanned and when we find issues in the scans there is currently no immediate way to find their source code repos without the document we are sharing.

The other (minor) change we are proposing is adding "Local" to the non-public cloud implementation instructions for K8S.  We have seen that simply calling local K8S by the name of "K8S" has caused a lot of confusion with folks wondering what the difference is.

We suggest this be reviewed by @carmine

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/917)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview:

* https://deploy-preview-917--competent-brattain-de2d6d.netlify.com/docs/reference/images/
* https://deploy-preview-917--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-k8s/

File: content/docs/started/getting-started-k8s.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 +++
-title = "Kubeflow on Kubernetes"
+title = "Kubeflow on Local Kubernetes"
```
Comment (sarahmaddox) [inline content/docs/started/getting-started-k8s.md]: s/Local/On-prem

This page is currently our solution for on-premises installations, although the process currently documented can also be used for local installations. We have a PR under review that adds LDAP auth to the page, and we want to leave the option open to provide on-prem alternatives to the Arrikto solution at some point in the future.

We should probably explain at the top of the page that these instructions are intended primarily for on-premises installations, although the process currently documented can also be used for local installations.

File: content/docs/reference/images.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
++++
+title = "Dockerfile Locations"
+description = "Where to find the Dockerfiles for all of Kubeflow's images"
+weight = 10
++++
+
+| Image Name        | Dockerfile Location |
+| ------------- |---------------|
+| tf_operator      | <<https://github.com/kubeflow/tf-operator/tree/master/build/images/tf_operato>r> |
```
Comment (sarahmaddox) [inline content/docs/reference/images.md]: There are a few too many angle-brackets on this line and the next, which are causing rendering problems in the links. See preview:
https://deploy-preview-917--competent-brattain-de2d6d.netlify.com/docs/reference/images/

**Comment by sarahmaddox** [comment]
Thanks! This LGTM from a tech writing point of view, after a couple of small fixes.

**Comment by sarahmaddox** [comment]
@JavaDerek Please would you round up the CLA signatures as mentioned by the bot. There are four email addresses with commits registered in this PR that need a CLA signature from the committers.

File: content/docs/started/getting-started-k8s.md (hunk 1 above)
Comment (carmine) [reply content/docs/started/getting-started-k8s.md]: Consider dropping this file from the commit? It is unrelated to the main focus of docker images.

**Comment by sarahmaddox** [comment]
Closing this PR, as the repo index has already been published via another PR.
The discussion around the Kubeflow on-prem/local/Kubernetes page is ongoing and will end up in another PR.

/close

**Pull request closed without merging**